src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -55,9 +56,10 @@ private static boolean hasInitialSoftClip( final List<CigarElement> cigarElement
         if ( firstEle.getOperator() == CigarOperator.HARD_CLIP && itr.hasNext() ) {
             firstEle = itr.next();
         }
+        final int clipStart = firstEle.getLength() - MIN_SOFT_CLIP_LEN;
         return firstEle.getOperator() == CigarOperator.SOFT_CLIP &&
-                firstEle.getLength() >= MIN_SOFT_CLIP_LEN &&
-                isHighQualityRegion(read.getBaseQualities(), 0);
+                clipStart >= 0 &&
+                isHighQualityRegion(read.getBaseQualities(), clipStart);
```

Comment:
Almost.  We've always checked just MIN_SOFT_CLIP_LEN base qualities.  It's just the position of this window that changes for left soft-clips.  I had intended to check the quals adjacent to the clip -- as I do for the right soft-clips -- but was instead checking from the beginning of the read for left soft-clips.
<---------soft clip--------><--------aligned part of read------>
<----check these--->  (old, wrong way)
            <---check these---> (new, right way)
This was particularly bad for negative strand reads, because I was checking where you'd expect bad bases to lurk (at the beginning of the RC'd read).
Also, I now allow 3 low quality bases in the checked window rather than 0, as you observed.